lib/web3dart.dart

```diff
@@ -28,7 +30,9 @@ export 'src/core/amount.dart';
 export 'src/core/block_information.dart';
 export 'src/core/block_number.dart';
 export 'src/core/sync_information.dart';
+export 'src/core/eth_rpc_query/eth_rpc_query.dart';
```

Comment:
i added a direct export from the main library file, correct me if you think any other way to export this custom implementation is better